package.json

```diff
@@ -32,8 +32,8 @@
     "lib"
   ],
   "scripts": {
-    "test": "make test",
-    "postinstall": "cd node_modules/monk/node_modules/mongodb && npm uninstall bson && cd ../../../; cd node_modules/mongodb && npm uninstall bson"
+    "test": "mocha --harmony test/mongorito.test",
```

Comment:
It is however your project so your welcome to overrule me on this however this was causing me issues which is why I switched.